x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -609,8 +624,9 @@ export class RulesClient {
       muteAll: false,
       mutedInstanceIds: [],
       notifyWhen,
-      executionStatus: getRuleExecutionStatusPending(new Date().toISOString()),
-      monitoring: getDefaultRuleMonitoring(),
+      running: false,
+      executionStatus: getRuleExecutionStatusPending(lastRunTimestamp.toISOString()),
+      monitoring: getDefaultMonitoring(lastRunTimestamp.toISOString()),
```

Comment:
When a rule is created, before running, it looks like `monitoring` is set to this:

```
"monitoring": {
  "run": {
    "history": [],
    "calculated_metrics": {
      "success_ratio": 0
    },
    "last_run": {
      "timestamp": "2022-11-03T13:06:31.486Z",
      "metrics": {
      "duration": 0,
        "total_search_duration_ms": null,
        "total_indexing_duration_ms": null,
        "total_alerts_detected": null,
        "total_alerts_created": null,
        "gap_duration_s": null
      }
    }
  }
}
```

Should those initial duration be `null` since we don't know it? Also is setting `last_run.timestamp` to the current date confusing since it hasn't run yet?

I'm thinking of a case where a rule is created in an overloaded Kibana cluster, so it might take longer than expected for the task to get claimed and run. In the UI, this `last_run.timestamp` would show up and give the impression that the rule has already run? Maybe I'm overthinking :)

Reply:
That is a good point, having 0 for the duration is definitely misleading

As for the timestamp, we're essentially mirroring existing `executionStatus -> lastExecutionDate` field with the `lastRun -> timestamp` field. So I guess setting the timestamp during `create` was existing behavior but we could fix it for both of the existing behavior if it doesn't make sense.

Reply:
That make sense!